packages/vite/src/node/plugins/html.ts

```diff
@@ -367,7 +367,7 @@ export function buildHtmlPlugin(config: ResolvedConfig): Plugin {
               prop.name === 'style' &&
               prop.type === NodeTypes.ATTRIBUTE &&
               prop.value &&
-              prop.value.content.includes('url(') // only url(...) in css need to emit file
+              /url\s*\(.*\)/.test(prop.value.content) // only url(...) in css need to emit file
```

Comment:
Is `url (some-url.bla)` valid CSS? The space between `url` and `(`. I checked and it doesn't seem like so

Reply:
Sorry, I found that my build didn't come out. I thought something was wrong. I should have run down dev🥲